.github/workflows/test.yml

```
@@ -0,0 +1,18 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
```

```suggestion
      fail-fast: false
      matrix:
```

Makes it so Actions does not abort the run if one matrix entry fails but runs to completion regardless of failures.